content/06.discussion.md

```diff
@@ -54,6 +54,34 @@ In assessing the upper bound on the predictive performance of a deep learning mo
+
+#### Confidence calibration, confidence intervals, and out-of-distribution detection
+
+In classification tasks, confidence calibration is the problem of using classifier scores to predict class membership probabilities that match the true membership likelihoods. These membership probabilities can be used to assess the uncertainty associated with assigning the example to each of the classes. Guo et al [@tag:guo_calibration] observed that contemporary neural networks are poorly calibrated and provide a simple recommendation for calibration: temperature scaling, a single parameter special case of Platt scaling [@tag:platt_scaling]. In addition to confidence calibration, there is early work from Chryssolouris et al [@tag:Chryssolouris1996_confidence] that described a method for obtaining confidence intervals with the assumption of normally distributed error for the neural network. More recently, Hendrycks and Gimpel discovered that incorrect or out-of-distribution examples usually have lower maximum softmax probabilities than correctly classified examples, allowing for effective detection of misclassified examples [@tag:out_dist_baseline]. Liang et al used temperature scaling and small perturbations to further separate the softmax scores of correctly classified examples and the scores of out-of-distribution examples, allowing for more effective detection [@tag:temp_out_dist].
```

Comment:
et al.

Reply:
I went through the section and fixed the issue everywhere.